emcc.py

```diff
@@ -1203,7 +1203,7 @@ def check(input_file):
       if shared.Settings.WASM_BACKEND:
         # setjmp/longjmp and exception handling JS code depends on this so we
         # include it by default.  Should be eliminated by meta-DCE if unused.
-        shared.Settings.EXPORTED_FUNCTIONS += ['_setThrew']
+        shared.Settings.EXPORTED_FUNCTIONS += ['_setThrew', '_emscripten_builtin_malloc', '_emscripten_builtin_free']
```

Comment:
This doesn't look good - we shouldn't export functions without a reason. An exported function increases code size. Why is this here?